### PR TITLE
[interoperator]  Runtime configs

### PR DESCRIPTION
**Pull request by vivekzhere**
- Worker count as runtime time config
- Read from configmap
- Use client-go to fetch configmap
- Informers not setup before controllers

### CODE REVIEW

File: interoperator/pkg/internal/config/config.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		kubeConfig: kubeConfig,
+		clientset:  clientset,
+	}, nil
+}
+
+func (cfg *config) updateConfig() error {
+	var err error
+	cfg.configMap, err = cfg.clientset.CoreV1().
+		ConfigMaps(constants.ServiceFabrikNamespace).
+		Get(constants.ConfigMapName, metav1.GetOptions{})
```
Comment (amitmalav) [inline interoperator/pkg/internal/config/config.go]: Function name is confusing, why name it `updateConfig` when we are fetching config?

Comment (vivekzhere) [reply interoperator/pkg/internal/config/config.go]: Renamed

**Comment by vivekzhere** [comment]
> Can we also mention this information somewhere? Maybe create a new wiki page [here](https://github.com/cloudfoundry-incubator/service-fabrik-broker/wiki)

#605 updates the SF2.0 read me with the interoperator config details also

### Configs supported by Interoperator

The number of workers for the service instance controller and service binding controller can be configured using a config map. The name of the config map must be `interoperator-config` and the namespace must be `default`. Sample config map is:


```
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: interoperator-config
  namespace: default
data:
  instanceWorkerCount: "5"
  bindingWorkerCount: "10"
```

The current supported configs are
* instanceWorkerCount - number of workers for the service instance controller
* bindingWorkerCount - number of workers for the service binding controller

If the config map is updated, interoperator must be restarted for the configs like number of workers to take effect.

**Comment by amitmalav** [comment]
@vivekzhere Can you make travis pass and also please run dev pipeline once

**Comment by vivekzhere** [comment]
> 
> 
> @vivekzhere Can you take care of creating the ConfigMap in the same NS as of the pods?

Reads the configmap from the same namespace as pods. If the namespace details is not there, reads from default namespace

**Pull request merged**